cmd/topaz-db/cmd/init.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	eds "github.com/aserto-dev/go-edge-ds"
+	"github.com/aserto-dev/go-edge-ds/pkg/directory"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+func (cmd *InitCmd) Run() error {
+	if fi, err := os.Stat(cmd.DBFile); err == nil {
+		if fi.IsDir() {
+			return fmt.Errorf("%s is a directory", cmd.DBFile)
+		}
+		return fmt.Errorf("%s already exists", cmd.DBFile)
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
I don't think this would handle SIGTERM and SIGKILL correctly.
I think we want the `main()` func to create a context that responds to signals and pass it down to subcommands.
```go
// in main.go
func main() {
   ...

   ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt)
   defer stop()

   // pass the context to subcommands
   if err := kongCtx.Run(ctx); err != nil {
       kongCtx.FatalIfErrorf(err)
   }
}
```

Reply:
fixed